src/LobbyClient.hs

```diff
@@ -22,30 +22,31 @@ clientMain api = do
   createBootstrapTemplate "Hastings"
   createChangeNickNameDOM api
   createChatDOM api
-  createLobbyDOM api
+  createLobbyDOM api newGameAPI
```

Comment:
The original thought as to why not (earlier it was created when creating a game) was that you can have different games, and then a different GameAPI would be required depending on the game. However, in the current state, I will change it so that there is only one GameAPI and then a generalisation will have to take place later on.